### PR TITLE
*: rephrase quickstart section of README

### LINKED ISSUE #9937: document and recommend try TiKV with TiUP
Update README to document and recommend try TiKV with TiUP, both start a release version or binary version built from source.

I'll try to take care of it this week.

### PR DESCRIPTION
**Pull request by tisonkun**
Signed-off-by: tison <wander4096@gmail.com>

### What problem does this PR solve?

Issue Number: close #9937 

Problem Summary:

The quickstart section of README, "Try TiKV", is out of date.

### What is changed and how it works?

Update quickstart section of README.

1. Mention & recommend TiUP
2. Mention running with binaries
3. Link to production-like cluster deployment and how to build from source.

### Release note

- No release note

### CODE REVIEW

**Review by andylokandy — APPROVED** [review]

**Comment by andylokandy** [comment]
Seems nice, good job!

**Comment by tisonkun** [comment]
thanks for your review @andylokandy !

**Comment by gregwebs** [comment]
Thanks for trying to make things easier for everyone! The only issue with this is that tiup is not a TiKV org project. So it may be best to have 2 separate docs, one for tiup, and one without. We can still suggest the tiup approach as the easiest.

**Review by zhongzc — APPROVED** [review]

**Comment by tisonkun** [comment]
> Thanks for trying to make things easier for everyone! The only issue with this is that tiup is not a TiKV org project. So it may be best to have 2 separate docs, one for tiup, and one without. We can still suggest the tiup approach as the easiest.

Initially I think of link TiUP document on [this page](https://docs.pingcap.com/tidb/stable/quick-start-with-tidb/#deploy-a-local-test-environment-using-tiup-playground) but is defeated by the temptation of inline it.

@gregwebs Do you think that we set a link to the page mentioned above a good idea?

**Comment by tisonkun** [comment]
> Thanks for trying to make things easier for everyone! The only issue with this is that tiup is not a TiKV org project. So it may be best to have 2 separate docs, one for tiup, and one without. We can still suggest the tiup approach as the easiest.

BTW I don't think TiUP is not a TiKV org project a problem. Most document recommends install software via Homebrew on MacOS even if it is not a standard pkg manager. TiUP is the (currently) recommended tool under maintenance and looks great.

**Comment by gregwebs** [comment]
I like linking to that page. It should be kept up to date whereas the docs in the TiKV repo have not always been that well maintained.

**Comment by tisonkun** [comment]
> I like linking to that page. It should be kept up to date whereas the docs in the TiKV repo have not always been that well maintained.

Make sense. Will do today.

**Comment by tisonkun** [comment]
> I like linking to that page. It should be kept up to date whereas the docs in the TiKV repo have not always been that well maintained.

@gregwebs done.

File: README.md [hunk 1]
```diff
@@ -73,30 +73,67 @@ You can see the [TiKV Roadmap](docs/ROADMAP.md).
...
 When a node starts, the metadata of the Node, Store and Region are recorded into PD. The status of each Region and Store is reported to PD regularly.
 
-## Try TiKV
+## Quick start
 
-TiKV was originally a component of [TiDB](https://github.com/pingcap/tidb). To run TiKV you must build and run it with PD, which is used to manage a TiKV cluster. You can use TiKV together with TiDB or separately on its own.
+### Deploy a playground with TiUP
 
-We provide multiple deployment methods, but it is recommended to use our Ansible deployment for production environment. The TiKV documentation is available on [TiKV's website](https://tikv.org/docs/4.0/concepts/overview/).
+The most quick way for trying out TiKV with TiDB is using TiUP, a component manager for TiDB.
```
Comment (gregwebs) [inline README.md:80]:
```suggestion
The most quickest to try out TiKV with TiDB is using TiUP, a component manager for TiDB.
```

**Review by gregwebs — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (tisonkun) [reply README.md:80]: the quickest?

**Pull request merged**